app/views/lookup/new.html.erb

```diff
@@ -1,10 +1,12 @@
 <%= display_header title: "Tag content", breadcrumbs: ["Tag content"] %>
 
-<%= form_for @lookup, url: find_by_slug_path do |f| %>
+<%= simple_form_for @lookup, url: find_by_slug_path do |f| %>
   <div class="form-group">
-    <%= f.text_field :base_path,
-      class: 'form-control',
-      label: "Path or URL of a page on GOV.UK (eg. /book-driving-test)" %>
+    <%= f.input :base_path,
+      label: "Path or URL of a page on GOV.UK (eg. /book-driving-test)",
```

Comment:
This label doesn't work? 

<img width="1175" alt="screen shot 2016-08-09 at 11 46 49" src="https://cloud.githubusercontent.com/assets/233676/17513793/fa4d907e-5e26-11e6-8173-247807985aaf.png">